### ISSUE TITLE
Slow processing decide scores

### ISSUE THREAD

**Issue by TMondain** [BiologicalRecordsCentre/DECIDE_adaptivesampling#9]
Manipulating raster layers is slow, especially if we want to load in the data for each species separately. I think that saving things as empty rasters and .csv (.fst?) files will make it quicker to load things into R.

Also, need to try using arrays rather than raster stacks when combining scores for different species as I think this could speed up processing time. E.g. if we want to combine the decide scores using a weighted average function, using arrays as a replacement for raster stacks *could* be quicker.

**Comment by AugustT** [comment]
@TMondain is this still a problem, if so could we move to an .rds approach?

**Comment by TMondain** [comment]
Yes, this is. It will be problematic if we want to be able to create a DECIDE score dependent on the species that are flying at that time of year. It could work with a .rds approach but not too sure how because raster package writes everything to a temporary directory. You can read all of the data into memory with raster::readALL() but might run into memory issues with the larger raster files. That's why was wondering about storing everything as csv files and having an empty raster to fill. I just haven't had time to think about it properly

**Comment by AugustT** [comment]
Probably challenge for DECIDE to tackle over the winter, ready for year 2

**Comment by TMondain** [comment]
agreed
